Configuration/StandardSequences/python/EventInterpretation.py

```diff
@@ -1,3 +1,4 @@
 EventInterpretation = {
-    'top' : 'CommonTools.ParticleFlow.EITopPAG_cff'
+    'top' : 'CommonTools.ParticleFlow.EITopPAG_cff',
+    'MiniAODfromMiniAOD' : 'PhysicsTools.PatAlgos.slimming.MiniAODfromMiniAOD_cff'
```

Comment:
it's OK for me in a short term, as long as this stays for some private recipes.
If  MiniAODfromMiniAOD turns into an official workflow (with central production implied), then the steps better be defined appropriately.